cumulus/parachains/runtimes/glutton/glutton-westend/src/lib.rs

```diff
@@ -236,7 +236,7 @@ impl pallet_aura::Config for Runtime {
 	type DisabledValidators = ();
 	type MaxAuthorities = ConstU32<100_000>;
 	type AllowMultipleBlocksPerSlot = ConstBool<true>;
-	type SlotDuration = ConstU64<2000>;
+	type SlotDuration = ConstU64<SLOT_DURATION>;
```

Comment:
I don't understand why we need to make this change. Why not run it with the slot based collator ? 

Should do 6s blocks just fine as is.